instrumentation/spring/spring-kafka-2.7/javaagent/build.gradle.kts

```diff
@@ -22,18 +22,46 @@ dependencies {
 
   testInstrumentation(project(":instrumentation:kafka:kafka-clients:kafka-clients-0.11:javaagent"))
 
-  testImplementation("org.testcontainers:kafka")
+  testImplementation(project(":instrumentation:spring:spring-kafka-2.7:testing"))
 
   testLibrary("org.springframework.boot:spring-boot-starter-test:2.5.3")
   testLibrary("org.springframework.boot:spring-boot-starter:2.5.3")
 }
 
+testing {
+  suites {
+    // CompletableFuture behaves differently if ForkJoinPool has no parallelism
```

Comment:
I didn't follow this comment?

Reply:
🙈 
I copy pasted the suites setup from the executors instrumentation, looks like I forgot to delete that one line 😂